apps/users/models/user/_auth.py

```diff
@@ -137,8 +137,8 @@ def set_ssh_key(self, public_key, private_key, **kwargs):
         if self.can_update_ssh_key():
             from authentication.models import SSHKey
             SSHKey.objects.create(
-                public_key=public_key, private_key=private_key, user=self, name=kwargs.get('name'),
-                comment=kwargs.get('comment'), is_active=kwargs.get('is_active')
+                public_key=public_key, private_key=private_key, user=self, name=kwargs.get('name', ''),
+                comment=kwargs.get('comment', ''), is_active=kwargs.get('is_active')
             )
             post_user_change_password.send(self.__class__, user=self)
 
```

Comment:
There's only one line of code with two occurrences of the same function that seems to be unnecessary and could've been merged together:

```
if self.can_update_ssh_key():
```